<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar::vertical{
background: none;
border-style: outset;
border-width: 1px;
border:none;
width:7px;
border-radius: 3px;
}

QScrollBar::handle:vertical {
background-color: #808080;
border-style: outset;
border-width: 1px;
border-radius: 3px;
width:7px;
}

QScrollBar::sub-line:vertical {
	border: none;
      background: none;

}


/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
		border: none;
      background: none;
}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}
QHeaderView::section
{

    background-color: #808080;
	color:#F5F5DC;
    padding: 4px;
    font:bold 14px;

}
QHeaderView::section:checked
{
    background-color:FFFFFF;
	
    
}
QTableView
{
    font:bold 14px;

}
QTableView::item
{
	background-color: #808080;
	color:#F5F5DC;
}
QTableView::item::selected
{
	background-color: #464646;
    
}

QTableView QTableCornerButton::section
{
    background-color: #808080;
    
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="resultTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#F8F8FF</string>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="visualizationFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>50</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputline">
         <property name="styleSheet">
          <string notr="true">background-color: #F8F8FF;
color:#808080;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 11px;
padding: 5px</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>167772</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="scrollBarStyle" stdset="0">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>699</width>
            <height>162</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="4" column="3">
            <widget class="QRadioButton" name="radioDescend">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>Sort in descending order</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="algoselector">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" alignment="Qt::AlignHCenter">
            <widget class="QCheckBox" name="checkBoxTime">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QRadioButton" name="radioAscend">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>Sort in ascending order</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" alignment="Qt::AlignHCenter">
            <widget class="QCheckBox" name="checkBoxMemory">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>Number of comparisons</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="genRandInputBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>generateRandombtn</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="undoRemoveRow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>undoRemoveRow</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="removeRowBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
             </property>
             <property name="text">
              <string>removeRow</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="visualizebtn">
         <property name="styleSheet">
          <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
         </property>
         <property name="text">
          <string>Visualize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnrun">
         <property name="styleSheet">
          <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
         </property>
         <property name="text">
          <string>Run algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hidebtn">
         <property name="maximumSize">
          <size>
           <width>114</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 11px;
padding: 5px</string>
         </property>
         <property name="text">
          <string>Hide visualization</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitbtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>9</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
